core/frontend/src/components/common/ErrorDialog.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-dialog 
+    v-model="show_dialog"
+    width="500"
+    transition="dialog-bottom-transition"
+  >
+    <v-card>
+      <v-card-title> Error </v-card-title>
+      <v-card-text> {{ error_message }} </v-card-text>
+      <v-card-actions>
+        <v-spacer />
+        <v-btn
+          text
+          @click="ignoreError"
+        >
+          Ignore this error
+        </v-btn>
+        <v-btn
+          text
+          @click="closeError"
+        >
+          Close
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { getModule } from 'vuex-module-decorators'
+import CommonStore from '@/store/common'
+import { Error } from '@/types/common'
+
+const common_store: CommonStore = getModule(CommonStore)
+
+export default Vue.extend({
+  name: 'Error',
+  computed: {
+    error (): Error | null {
+      return common_store.current_error
+    },
+    error_message (): string {
```

Comment:
`error` already provides the abstract for message, we should rely on the components that already exist. 
An abstraction just to access a member of a function that already return sounds as not a good interface practice.

Reply:
Will check on using it.